packages/rest/src/router/metadata.ts

```diff
@@ -166,14 +224,20 @@ function resolveControllerSpec(
  * @param constructor Controller class
  */
 export function getControllerSpec(constructor: Function): ControllerSpec {
-  let spec = Reflector.getOwnMetadata(API_SPEC_KEY, constructor);
+  const scope = Reflector.getOwnMetadata(CONTROLLER_SCOPES_KEY, constructor);
+  const key = scope ? `${API_SPEC_KEY}:${scope}` : API_SPEC_KEY;
```

Comment:
I still prefer to use a fixed metadata key and maintain the value as `Map<serverName, spec>` if necessary.

Reply:
Sorry, was an interim commit; wanted to validate the approach of using another metadata field to differentiate between them.

I can make it multi-server soon.